app/migrations/Version20210525155405.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Mautic\Migrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use Doctrine\Migrations\Exception\SkipMigration;
+use Mautic\CoreBundle\Doctrine\AbstractMauticMigration;
+
+final class Version20210525155405 extends AbstractMauticMigration
```

Comment:
Is there any reason why this migration for the reports table is extra? Can the reports table be added to the other [migration](https://github.com/mautic/mautic/pull/14558/files#diff-a52f6525937f5bbb9995137906cd14c850fc7ea35d48fa334e4a4b090b1b2b7cR17) that handles the rest of the tables?